### DISCUSSION TITLE
Misplace arrow popover

### DISCUSSION THREAD

**Discussion by nguyenlambuu** [primefaces/community#3265] [PrimeNG] [answered]
PrimeNG v19 is misplaced the arrow
![IMG_7800](https://github.com/user-attachments/assets/0bd90f15-461e-46e0-b5a0-bc876800d91e)

It should be like this

![IMG_7801](https://github.com/user-attachments/assets/22555143-6e60-4545-a98c-e64b6328b065)

**Comment by manux54** [comment, marked as answer]
It seems to be partially implemented, but not completed. There are two css variables that seems to be set up to position the arrow: --p-popover-arrow-offset (the offset from the left side) and --overlayArrowLeft an additional offset when the popover is pushed to fit on screen. But only --p-popover-arrow-offset appears to be used. Currently:
`
.p-popover:after, .p-popover:before {
  left: var(--p-popover-arrow-offset);
}
`
In my own css I put 
`
.p-popover:after, .p-popover:before {
  left: calc(var(--p-popover-arrow-offset) + var(--overlayArrowLeft));
}
`
and it works fine.

**Comment by nguyenlambuu** [comment]
Quick test and see it works, thanks a lot @manux54 
<img width="1129" alt="image" src="https://github.com/user-attachments/assets/8dcaa73d-bba3-49ff-805b-99edc2021ecd" />
